package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nova-tables",
-  "version": "1.8.3",
+  "version": "1.8.4",
```

Comment:
If nothing else this is the wrong version increment. This change is not backwards compatible as the http instance can no longer be configured. This must at least be version 1.9.0.

Reply:
updated